index.html

```diff
@@ -6,6 +6,9 @@
     <title>AOD</title>
     <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&family=Roboto+Slab:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="css/reset.css" type="text/css">
+    <!-- SLICK -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.9.0/slick.min.css" type="text/css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.9.0/slick-theme.min.css">
```

Comment:
dodaj atrybut type do tego linku, tak ja to zrobiles powyzej

Reply:
pkt 3) przepraszam moje nie dopatrzenie i "skleroza" ustawialem na rozdzielczosci 768px i zapomnialem ze jesli na nastepnej rozdzielczosci tego nie zmienie to odziedziczy wlasnie z poprzedniej.